limitador-server/src/main.rs

```diff
@@ -290,12 +295,37 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 .max_level_hint()
                 .unwrap_or(LevelFilter::ERROR)
         });
-        let builder = if level >= LevelFilter::DEBUG {
-            tracing_subscriber::fmt().with_span_events(FmtSpan::CLOSE)
+        let fmt_layer = if level >= LevelFilter::DEBUG {
+            tracing_subscriber::fmt::layer().with_span_events(FmtSpan::CLOSE)
         } else {
-            tracing_subscriber::fmt()
+            tracing_subscriber::fmt::layer()
+        };
+
+        if !config.tracing_host.is_empty() {
+            let tracer = opentelemetry_otlp::new_pipeline()
+                .tracing()
+                .with_exporter(
+                    opentelemetry_otlp::new_exporter()
+                        .tonic()
+                        .with_endpoint(format!("rpc://{}", config.tracing_address()))
+                        .with_protocol(Protocol::Grpc),
```

Comment:
I think I'd want to align that with authorino's way of configuring...

Reply:
I've had a look and this `.with_protocol` doesn't actually effect the behaviour. Either we use `.tonic()` and it's grpc, or we use `.http()` and provide our choice of http client. For now I've moved to a `--tracing-endpoint` option which contains scheme the same as authorino, but it won't use http if provided unless we check for the provided scheme and configure an http client